src/main/java/it/gov/pagopa/ranking/dto/mapper/InitiativeBuild2ConfigMapper.java

```diff
@@ -34,6 +34,7 @@ public InitiativeConfig apply(InitiativeBuildDTO initiativeBuildDTO) {
                 .rankingStatus(RankingStatus.WAITING_END)
                 .size(calculateSize(initiativeBuildDTO))
                 .rankingFields(retrieveRankingFieldCodes(automatedCriteriaList))
+                .organizationName(initiativeBuildDTO.getOrganizationName())
```

Comment:
let's keep fields ordered in order to find them more easily, put this after organizationId field